Neos.Flow/Classes/Mvc/Routing/Routes.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Neos\Flow\Mvc\Routing;
+
+use Neos\Flow\Annotations as Flow;
+use Traversable;
+
+/**
+ * @Flow\Proxy(false)
+ */
+final class Routes implements \IteratorAggregate
+{
+    /**
+     * @var array<int, Route>
+     */
+    private readonly array $routes;
+
+    private function __construct(
+        array $routes
+    ) {
```

Comment:
```suggestion
    private function __construct(
        Route ... $routes
    ) {
```
for strict typing (calling side now needs adjustments aswell (`new self(...$routesArray)`) )